cli/cmd/cluster.go

```diff
@@ -144,26 +144,31 @@ var _clusterUpCmd = &cobra.Command{
 
 		clusterConfigFile := args[0]
 
-		envExists, err := isEnvConfigured(_flagClusterUpEnv)
-		if err != nil {
+		if _, err := docker.GetDockerClient(); err != nil {
 			exit.Error(err)
 		}
-		if envExists {
-			if _flagClusterDisallowPrompt {
-				fmt.Printf("found an existing environment named \"%s\", which will be overwritten to connect to this cluster once it's created\n\n", _flagClusterUpEnv)
-			} else {
-				prompt.YesOrExit(fmt.Sprintf("found an existing environment named \"%s\"; would you like to overwrite it to connect to this cluster once it's created?", _flagClusterUpEnv), "", "you can specify a different environment name to be configured to connect to this cluster by specifying the --configure-env flag (e.g. `cortex cluster up --configure-env prod`); or you can list your environments with `cortex env list` and delete an environment with `cortex env delete ENV_NAME`")
-			}
-		}
 
-		if _, err := docker.GetDockerClient(); err != nil {
+		accessConfig, err := getNewClusterAccessConfig(clusterConfigFile)
+		if err != nil {
 			exit.Error(err)
 		}
 
-		accessConfig, err := getNewClusterAccessConfig(clusterConfigFile)
+		envName := _flagClusterUpEnv
+		if envName == "" {
+			envName = accessConfig.ClusterName
+		}
```

Comment:
note to reviewer: this is a weird diff since the code was rearranged, these are the only lines that were added